scripts/webp/0.5.1/script.sh

```diff
@@ -19,6 +19,11 @@ function mason_load_source {
 
 function mason_compile {
     export CFLAGS="${CFLAGS:-} -Os"
+
+    if [[ ${MASON_PLATFORM_VERSION} == "cortex_a9" ]]; then
+        CFLAGS="${CFLAGS:-} -mfloat-abi=softfp -mfpu=neon"
```

Comment:
Don't think so. This directly affects the floating point precision, so it should be evaluate case by case.